app/src/main/res/values-es/strings.xml

```diff
@@ -1,5 +1,208 @@
+  <string name="title_slider2">Mis datos</string>
+  <string name="title_slider3">Selección de estación</string>
+  <string name="title_slider4">Tomar una foto</string>
+  <string name="title_slider6">Compartir una foto</string>
+  <string name="title_slider7">Notificar de estaciones cercanas</string>
+  <string name="title_slider8">El chat</string>
+  <string name="title_slider9">Seleccionar pais</string>
+  <string name="missing_licence">Najo que licencia quiere subir sus fotos?</string>
+  <string name="missing_photoOwner">Por favor confirme que son sus fotos.</string>
+  <string name="missing_nickname">Por favor ingrese su nickname</string>
+  <string name="missing_email_address">Por favor ingrese una dirección de correo valido</string>
+  <string name="missing_linking">Por favor eliga si desea ser enlazado</string>
+  <string name="missing_link">Por favor ingrese una URL valida si desea ser enlazado</string>
+  <string name="upload_conflict">Su foto ha sido subida,pero puede haber un conflicto. O ha subido la misma foto dos veces, u otro usuario ya subio una fotografia.</string>
+  <string name="select_photo">Selección de foto</string>
+  <string name="station_photo">Foto de la estación</string>
+  <string name="menu_title_stations_map">Mapa de la estacion</string>
+  <string name="menu_photo_filter">Filtrar con/sin foto</string>
+  <string name="loading_stations">Cargar estaciones</string>
+  <string name="search_nickname_hint">Buscar fotógrafo </string>
+  <string name="cc0_needed">Por favor acepte CC0, para que podamos usar su foto sin problemas.</string>
+  <string name="update_available">Nuevas fotos de la estación disponibles, actualizar?</string>
+  <string name="button_cancel_text">Cancelar</string>
+  <string name="update_manual">Manual</string>
+  <string name="update_notify">Preguntar</string>
+  <string name="update_automatic">Automaticamente</string>
+  <string name="tvUpdatePolicy">Subir los datos de la estación</string>
+  <string name="send">Enviar</string>
+  <string name="logo_train">logo de una pequeña locomotora</string>
+  <string name="license_info_not_readable">Información de la licencia ilegible por el momento</string>
+  <string name="station_not_found">Estación no encontrada</string>
+  <string name="last_update_at">Ultima vez actualizado en: </string>
+  <string name="station_update_failed">Actualización de estacion fallida:</string>
+  <string name="stationId">StationNbr: </string>
+  <string name="need_to_allow_location_access">Necesita acceder a dar permisos locales</string>
+  <string name="my_position">Mi posición actual es : </string>
+  <string name="chat_notification_enabled">Ha activado satisfactoriamente las notificaciones del chat</string>
+  <string name="chat_notification_disabled">Ha desactivado satisfactoriamente las notificaciones del chat</string>
+  <string name="unable_to_create_folder_structure">Fallo al crear estructura de direcciones/string>
```

Comment:
@troyodk, I have found it! You have missed `<` before `/string` here. If you fix it, all will be good.